### PR TITLE
2557 Add map overlays for covid vaccine tracking

### PR DESCRIPTION
**Pull request by IgorNadj**
### Issue #: 
- [2557](https://github.com/beyondessential/tupaia-backlog/issues/2557)
- [2558](https://github.com/beyondessential/tupaia-backlog/issues/2558)

### Screenshots:
https://github.com/beyondessential/tupaia-backlog/issues/2557#issuecomment-807954402

### CODE REVIEW

File: packages/database/src/migrations/20210326034445-AddCovidVaccineOverlays-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    code: 'COVID19_Nauru',
+  },
+}
+
+const MAP_OVERLAY_TEMPLATES = {
+  Total_Number_Of_People_1st_Dose_Covid_Vaccine: {
+    name: 'Total number of people received 1st dose of COVID-19 vaccine',
+    userGroup: 'COVID-19',
+    dataElementCode: 'COVIDVac4',
+    measureBuilderConfig: {"entityAggregation": {"aggregationType": "REPLACE_ORG_UNIT_WITH_ORG_GROUP", "dataSourceEntityType": "village", "aggregationEntityType": "village"}},
```
Comment (billli0) [inline packages/database/src/migrations/20210326034445-AddCovidVaccineOverlays-modifies-data.js:44]:
```suggestion
    measureBuilderConfig: {"entityAggregation": {"aggregationType": "REPLACE_ORG_UNIT_WITH_ORG_GROUP", "dataSourceEntityType": "district", "aggregationEntityType": "village"}},
```

Here we need to change it for Nauru and Fiji. See [comment from Slack](https://beyondessential.slack.com/archives/C01BXE4KP5K/p1616969807002900)

File: packages/database/src/migrations/20210326034445-AddCovidVaccineOverlays-modifies-data.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    measureBuilder: 'sumAllPerOrgUnit',
+    presentationOptions: {"scaleType": "neutral", "displayType": "spectrum", "measureLevel": "Village"},
+  },
+  Total_Number_Of_People_2nd_Dose_Covid_Vaccine: {
+    name: 'Total number of people received 2nd dose of COVID-19 vaccine',
+    userGroup: 'COVID-19',
+    dataElementCode: 'COVIDVac8',
+    measureBuilderConfig: {"entityAggregation": {"aggregationType": "REPLACE_ORG_UNIT_WITH_ORG_GROUP", "dataSourceEntityType": "village", "aggregationEntityType": "village"}},
```
Comment (billli0) [inline packages/database/src/migrations/20210326034445-AddCovidVaccineOverlays-modifies-data.js:52]: Same here.

**Review by billli0 — CHANGES_REQUESTED** [review]
Hi Igor, very clean code. There is a bit requirement changes from Slack and didn't comment in the cards, could you do the update?

File: packages/database/src/migrations/20210326034445-AddCovidVaccineOverlays-modifies-data.js (hunk 1 above)
Comment (IgorNadj) [reply packages/database/src/migrations/20210326034445-AddCovidVaccineOverlays-modifies-data.js]: Thanks, changed to district level, see screenshot

File: packages/database/src/migrations/20210326034445-AddCovidVaccineOverlays-modifies-data.js (hunk 2 above)
Comment (IgorNadj) [reply packages/database/src/migrations/20210326034445-AddCovidVaccineOverlays-modifies-data.js]: Updated

**Review by billli0 — APPROVED** [review]

**Pull request merged**